packages/devextreme/scss/widgets/base/splitter/_index.scss

```diff
@@ -5,6 +5,7 @@
 
   .dx-splitter-item {
     flex: 1 0 0px;
+    overflow: hidden;
```

Comment:
With this style vertical splitter collapse their panes, if the height of parent splitter is not declared.

Reply:
I suggest try to use 'min-width: 0' property to resolve issue what you mentioned earlier.